app/move/fields/create.js

```diff
@@ -124,6 +124,15 @@ const dateField = {
   },
 }
 
+const items = {
```

Comment:
`items` names feels a bit ambiguous given how large this large file now and how many other shared variables we have. Items is a property on many of these fields and this option wouldn't apply to move of them.

What about just creating a variable for the court appearance move type item:

```javascript
moveTypeCourtAppearance: {
   ...
}
```

and inlcuding like this:

```javascript
items: [{ ...moveTypeCourtAppearance }],
```